.travis.yml

```
@@ -52,5 +52,17 @@ jobs:
         - BROWSERS="FirefoxHeadless"
       addons:
         firefox: latest-esr
+    - name: "Browser tests: browsertack tests"
```

A typo + different casing:
```suggestion
    - name: "Browser tests: BrowserStack tests"
```